### ISSUE TITLE
`unit-test-missing` false positive when the main object is a concrete (non-abstraction) object

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#952]
**What happens:** The `unit-test-missing` lint reports a warning for EO files where the main object is a concrete object. For example:

```eo
+architect yegor256@gmail.com
+home https://github.com/objectionary/eo
+version 0.0.0
+spdx SPDX-FileCopyrightText: Copyright (c) 2016-2026 Objectionary.com
+spdx SPDX-License-Identifier: MIT
+unlint redundant-object

# The `not a number` object is an abstraction
# for representing undefined or unrepresentable numerical results.
number 7F-F8-00-00-00-00-00-00 > nan
```

Here `nan` is a concrete object — it is a fixed value bound to `number`, with no free parameters, no behavior to vary, and nothing a unit test can exercise. There is simply no test to write.

The only available workaround is to add `+unlint unit-test-missing`, which is manual overhead for an inherently untestable structure.

**What should happen:** The `unit-test-missing` lint should automatically skip files where the main object is a concrete (non-abstraction) object, without requiring a manual `+unlint` directive.

**Comment by volodya-lombrozo** [comment]
@rultor release, tag is `0.2.4`, title is `0.2.4`.
